main.py

```diff
@@ -74,17 +75,12 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            user_contributions = user_services.get_user_contributions(
+            user_settings = user_services.get_user_settings(
                 self.user_id)
-
-            # 'Creator' is a user who has created or edited an exploration.
-            user_is_creator = user_contributions and (
-                user_contributions.created_exploration_ids or
-                user_contributions.edited_exploration_ids)
-            if user_is_creator:
+            if user_settings.default_dashboard == constants.DASHBOARD_TYPE_CREATOR: # pylint: disable=line-too-long
```

Comment:
We really should try to avoid using pylint pragmas where possible -- they aren't supposed to become a "standard thing". Here, you might consider declaring a new var above this like default_dashboard_is_creator and then doing `if default_dashboard_is_creator` instead.